### DISCUSSION TITLE
Image preview of the tree species

### DISCUSSION THREAD

**Discussion by vogelino** [technologiestiftung/giessdenkiez-de#233] [Ideas]
It would help it feel more real if I could click on a tree and have a preview image of the species of the tree I just selected. 

**Example:**
![species-pics](https://user-images.githubusercontent.com/2759340/105684450-44405f80-5ef5-11eb-9f86-410e758a1d8f.png)

**Comment by ff6347** [comment]
I would totally dig this. Would be nice to use the app to learn a little about trees. How they look and so on
